### PR TITLE
[Compute] `az vmss update`: Add new parameter `--force-deletion` to support force delete VMSS

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
**Description**

Support forceDeletion of in VMSS ScaleIn Policy in Azure CLI  https://github.com/Azure/azure-cli/issues/20301

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by jiasli** [comment]
@zhoxing-ms, please help with this PR.

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -678,6 +678,7 @@ def load_arguments(self, _):
                    help='Enable terminate notification')
         c.argument('ultra_ssd_enabled', ultra_ssd_enabled_type)
         c.argument('scale_in_policy', scale_in_policy_type)
+        c.argument('force_deletion', arg_type=get_three_state_flag(), is_preview=True, help='This property allow you to specify if virtual machines chosen for removal have to be force deleted when a virtual machine scale set is being scaled-in.')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py:681]: `action='store_true'` should be a more appropriate option than `get_three_state_flag()`. Moreover, other similar parameters `force_update` also use `action='store_true'`, they should be consistent

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py [hunk 2]
```diff
@@ -2574,6 +2574,9 @@ def test_vmss_scale_in_policy(self, resource_group):
         self.cmd('vmss update -g {rg} -n {vmss} --scale-in-policy OldestVM', checks=[
             self.check('scaleInPolicy.rules[0]', 'OldestVM')
         ])
+        self.cmd('vmss update -g {rg} -n {vmss} --force-deletion true', checks=[
+            self.check('scaleInPolicy.forceDeletion', True)
+        ])
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:2579]: Why is the yaml file not re-recorded after this test is modified?

Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:2579]: There is a problem with the service https://github.com/Azure/azure-cli/issues/20301#issuecomment-986541263

**Comment by yonzhan** [comment]
Compute

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**